2019 eCQM uuid updates

saquino0827 opened the pull request:
QPPSF-4791 

#### Motivation for change

The 2019 eCQM uuid's are being updated to match the 2019 implementation guide: https://ecqi.healthit.gov/system/files/2019_CMS_QRDA_III_Eligible_Clinicians_and_EP_IG-508.pdf

#### What is being changed

Mostly eCQM uuid's were changed. `scripts/measures/2019/enrich-measures-data.js` to store data into `util/measures/' + currentYear + '/`. It was also changed to handle scenarios where strata.name was not provided.

#### Release checklist:
Tasks that must be done prior to merging this PR, including testing.

* [x] [Package version updated](https://github.com/CMSgov/qpp-measures-data/blob/master/CONTRIBUTING.md#versioning-publishing-and-creating-new-releases)
* [x] Documentation updated
* [x] Unit tests added/passing
* [x] Verified working locally

##### Associated JIRA tickets:
https://jira.cms.gov/browse/QPPSF-4791

saquino0827 commented:
For these for measure's `CMS135v7, CMS144v7, CMS157v7, CMS347v2`, please see the confluence page below for an explanation of why they have multiple strata but are single performance rates.:
https://confluence.cms.gov/pages/viewpage.action?spaceKey=QPPPRACTICE&title=QRDA+Conversion+from+Multiple+to+Single+Rate

Jeffreyher commented:
Hey, I just removed 'registry' from `submissionMethod` for eCQM's. Can you make sure your pr does not re-add these when it is merged?

saquino0827 commented:
@Jeffreyher  I removed the code that was adding `registry` as a submission method and now they are no longer being added. I also added some additional stratifier uuid's. These are not strata but a subset of uuid's under strata required for conversion tool validation.

saquino0827 commented:
@lamroger Whenever you have time, can you take a look at this? Thanks!

lamroger commented:
I'm out today. Could you ping someone in the qppa Slack channel? Thanks!

On Tue, Aug 20, 2019 at 8:21 AM Samuel Aquino <notifications@github.com>
wrote:

> @lamroger <https://github.com/lamroger> Whenever you have time, can you
> take a look at this? Thanks!
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/CMSgov/qpp-measures-data/pull/278?email_source=notifications&email_token=AAES2KQGKO6BLSBITEM55QDQFQDYXA5CNFSM4IGQT742YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD4WVKEY#issuecomment-523064595>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAES2KUG5NGEJPJNMAHVIS3QFQDYXANCNFSM4IGQT74Q>
> .
>
